utils/utils.py

```
@@ -0,0 +1,25 @@
+
+
+def start_squid_container(container_name='squid-proxy'):
+    """Start squid proxy container"""
+    subprocess.run(
+        [f'podman run -d --name {container_name} -p 3128:3128 -v $(pwd)/squid.conf:/etc/squid/squid.conf:Z ubuntu/squid'],
+        shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, encoding='utf-8')
+
+
+def clear_squid_logs(container_name='squid-proxy'):
+    """Restart squid container to clear logs before test"""
+    subprocess.run(['podman', 'restart', container_name], check=True)
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)